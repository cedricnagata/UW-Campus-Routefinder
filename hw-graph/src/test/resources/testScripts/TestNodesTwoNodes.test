# Create a graph
CreateGraph Graph

# Adds two nodes to the graph
AddNode Graph n1
AddNode Graph n2

# List the Nodes in the graph
ListNodes Graph