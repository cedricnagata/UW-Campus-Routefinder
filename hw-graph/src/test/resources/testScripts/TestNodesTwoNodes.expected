# Create a graph
created graph Graph

# Adds two nodes to the graph
added node n1 to Graph
added node n2 to Graph

# List the Nodes in the graph
Graph contains: n1 n2